src/phoenix/server/main.py

```diff
@@ -157,6 +160,12 @@ def _load_items(
             args=(traces, fixture_spans, simulate_streaming),
             daemon=True,
         ).start()
+        fixture_evals = get_evals_from_fixture(trace_dataset_name)
+        Thread(
+            target=_load_items,
+            args=(evals, fixture_evals, simulate_streaming),
+            daemon=True,
+        ).start()
```

Comment:
future thought: I might need to have a "dirty" bit to know when evals change so I know how to refetch due to the lack of subscriptions :(

Reply:
not sure how exactly. let's catch up later